db/schema.rb

```diff
@@ -476,7 +476,7 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.boolean "scanner_working"
-    t.index ["uploader_type", "uploader_id"], name: "index_malware_scan_results_on_uploader_type_and_uploader_id"
+    t.index ["uploader_type", "uploader_id"], name: "index_malware_scan_results_on_uploader"
```

Comment:
Is this related to this PR?

Reply:
This keeps automatically happening when you run rails migrations. Mark and I have seen it time and time again. I think it's an update to clamav that's doing it (I think).

Reply:
I think a discussion was to be had about it but has probably been forgotten. It's probably fine to leave in as it is autogenerated but might be worth a check (not sure what's the best way to do that other than everything works as expected).

Reply:
i think we can stop this happening by explicitly specifying the name - it's happening because the name is too long I think.

Reply:
I think this has been updated elsewhere now doing precisely that. So if you rebase I think this will then disappear from the PR anyway.